processor/error/_meta/fields.yml

```diff
@@ -2,24 +2,44 @@
   title: APM Error
   description: Error-specific data for APM
   fields:
+    - name: view errors 
```

Comment:
not too happy with the name of the field here. I know it's what is used now. I'm thinking if we should also have here some conventions like not using spaces in the future? This is something that would have to be done when creating the dashboard.

Reply:
I did not intend to change any outcome of the dashboards, so no namings, linkings etc. i just moved the current parts to the right places to enable automatic loading and automatic adapting of the index pattern (which was pretty outdated already). 

Hence, if you are unhappy with the naming, please bring this up with @simianhacker and the rest of the APM team.

Reply:
Definitively. Wasn't meant to deal with in this PR.

Reply:
I just realized that my former comment might have come off a bit harsh, this was not my intention.